config/manager/manager.yaml

```diff
@@ -34,8 +34,8 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 768Mi
+            memory: 1Gi
```

Comment:
should we make it higher?

Reply:
changed to 2Gi

Reply:
Wonder if we could test this to get a rough idea of the memory requirements? go-git has open issues for using lots of memory....